### PR TITLE
Flow for SCS services

### PR DESCRIPTION
**Pull request by quis**
## _Depends on https://github.com/alphagov/digitalmarketplace-utils/pull/94_

For SCS services, this pull request adds pages to:
- view a summary of a service
- view a page of fields for editing one aspect of a service

--

It doesn't:
- work with the drafts API (it just shows an existing service as if it were a draft)
- do anything with `POST`ed data
- show existing values for checkboxes and radios—need to think about how to pass this data through to the templates
- have proper fields for pricing or file uploads—going to work on these separately in the toolkit

…but that said I think it's better to get this merged and build on it, rather than try to do everything in one PR.

--

gov uk - the best place to find government services and information

### CODE REVIEW

**Comment by TheDoubleK** [comment]
`NameError: global name 'ContentBuilder' is not defined` at this line: https://github.com/alphagov/digitalmarketplace-supplier-frontend/blob/flow-for-scs-services/app/main/views/services.py#L283

File: app/main/views/services.py [hunk 1]
```diff
@@ -172,6 +169,81 @@ def update_section(service_id, section):
...
+
+    if "success":
+        return redirect(
+            url_for(".view_service_submission", service_id=service_id)
+        )
+    else:
+        return render_template(
+            "services/edit_section.html",
+            section=content.get_section(section),
+            service_data=service_data,
```
Comment (TheDoubleK) [inline app/main/views/services.py]: `service_data` is not a thing (i.e. "Unresolved reference 'servcie_data')

File: app/main/views/services.py [hunk 2]
```diff
@@ -172,6 +169,81 @@ def update_section(service_id, section):
...
+            service_id=service_id,
+            error=e.message,
```
Comment (TheDoubleK) [inline app/main/views/services.py]: `e` is not a thing here (i.e. "Unresolved reference 'e')

**Comment by TheDoubleK** [comment]
At this line: https://github.com/alphagov/digitalmarketplace-supplier-frontend/blob/flow-for-scs-services/app/main/views/services.py#L148

I get `AttributeError: 'ContentBuilder' object has no attribute 'get_question'`

**Comment by quis** [comment]
@TheDoubleK Addressed your comments in 20d636d.

File: app/main/views/services.py [hunk 3]
```diff
@@ -162,8 +159,8 @@ def update_section(service_id, section):
         except HTTPError as e:
             return render_template(
                 "services/edit_section.html",
-                section=content.get_section_filtered_by(section, service),
-                service_data=service,
+                section=content.get_section(section),
+                service_data=service_data,
```
Comment (TheDoubleK) [inline app/main/views/services.py]: Another one - this should be `service_data=posted_data`

File: app/templates/services/edit_section.html [hunk 4]
```diff
@@ -45,7 +45,7 @@
 
     </div>
   </div>
-  <form method="post" action="{{ url_for(".update_section", service_id=service_id, section=section.id) }}">
+  <form method="post" action="{{ url_for(post_to, service_id=service_id, section=section.id) }}">
```
Comment (TheDoubleK) [inline app/templates/services/edit_section.html]: `UndefinedError: 'post_to' is undefined`

**Comment by quis** [comment]
@TheDoubleK Addressed ALL your comments in 0dc3cf9.

**Comment by TheDoubleK** [comment]
Looks good now as a starter for 10 - so much will change as submissions gets built out that I don't think it's worth picking over with a fine toothcomb right now. Happy for this to be merged once the required Utils version exists and Travis is happy.

File: app/templates/services/edit_section.html [hunk 5]
```diff
@@ -60,7 +60,7 @@
     %}
       {% include "toolkit/button.html" %}
     {% endwith %}
-    <a href="{{ url_for(".edit_service", service_id=service_id) }}">Return to service</a>
+    <a href="{{ url_for(return_to, service_id=service_id) }}">Return to service</a>
```
Comment (TheDoubleK) [inline app/templates/services/edit_section.html]: Sorry... `UndefinedError: 'return_to' is undefined`

**Comment by quis** [comment]
@TheDoubleK Missing `return_to` added in 4dcd1cb.

**Pull request merged**